pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -974,6 +978,8 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 						prefix = "net"
 					} else if net.Genie != nil {
 						prefix = "eth"
+					} else if net.Tungstenfabric != nil {
+						prefix = "eth"
```

Comment:
we need a test in converter_test.go to ensure the correct domain xml is generated when a TungstenFabric network is in use.

Reply:
done:
https://github.com/kubevirt/kubevirt/pull/1956/commits/3865d166e9b73e6be4058db819f5658a4c18d911#diff-406f8a4e9665b5e52c190c5815dbacd2R1020